### DISCUSSION TITLE
[八卦]白祇重工之前发生什么了？

### DISCUSSION THREAD

**Discussion by Aris-Tendo** [share121/inter-knot#325] [其他]
![image](https://github.com/user-attachments/assets/a6b755ff-2047-4579-b5b1-9ac9039ae264)

在一个儿童节目上看到了白祇重工的嘉宾，主持人竟然当着白祇重工的面提到白祇重工数年前有过一个大丑闻。虽然这个主持人明显是竞争对手派来搞事的，可我还是有点在意！有人知道具体是怎么回事吗？

**Comment by cpu644** [comment]
最高端的商战 最朴素的方案

**Comment by Bambook-sth** [comment]
听说是前社长卷钱跑路了，看来不管是远景还是白祈重工都不是很能让人相信啊

**Comment by npldqt** [comment]
这地铁项目都多久了，这市长政绩还要不要了

**Comment by banyue121** [comment]
好像是旧都陷落那个时期的事情了，那个时候能接触到钱的人卷款跑路这种事还挺多的。

**Comment by nahida0-o** [comment]
楼主你这语气，我还以为现在地铁工程又出什么事了。
